docs/core/clone_module.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: default
+title: Cloning a module
+nav_order: 10
+parent: Core
+---
+
+# Cloning a module
+
+A (source) module instance can be cloned (copied) by the
+`cluster/clone-module` action. This action creates a new module instance
+(destination) that is equivalent to the source one.
+
+For example to create a clone/copy of instance `dokuwiki1` on node 1, run
+
+    api-cli run clone-module --data '{"module":"dokuwiki1","replace":false,"node":1}'
+
+The cluster **node** where the destination runs can be the same of the
+source instance or not; generally if the services provided by the instance
+do not require exclusive access to a particular system resource (i.e. bind
+a given TCP port number) there should be no limitation on running multiple
+instances of the same module on the same node. In the end, this kind of
+limitations must be managed by the module itself.
+
+In a nutshell, `clone-module`:
+
+- Creates a new module instance for the destination using `cluster/add-module`
+- Starts two parallel subtasks:
+  - `transfer-state` on the source instance
+  - `clone-module` on the destination instance
+- When both subtasks are succesfully completed and if a full **replace**
+  is required, the source instance is removed with `cluster/remove-module`
+- The destination instance has the same MODULE_UUID if replace is required
+
+Module must implement `transfer-state` and `clone-module` to correctly
```

Comment:
```suggestion
Module can implement `transfer-state` and `clone-module` to correctly
```
For me *can* is more right, the cloning process can work also correctly without some implementation by the module developer. Or al least we can use the verb *should*.